lib/classes/box_pallet_manager.rb

```diff
@@ -1,9 +1,11 @@
 class BoxPalletManager
```

Comment:
@abhinav098  All the logical operations are currently in the `concerns/operations` folder. Could you add this feature there ? We're trying to consolidate all the business logic in the same place
There is a `stock_operation.rb` which would work quite well for the box features.

The `BoxPalletManager` can still exist under the hood, but would like to have all the entry points in the same place :)

It populates the `Package::Operations` namespace. Meaning that adding methods there would allow us to do `Package::Operations.pack(package, box: box, quantity: 5)` (or something like that)

Reply:
sure @patrixr , thanks for the early feedback.

Reply:
Thanks ! Feel free to fiddle with the code 😄 

I'm not dead set on having operations as concerns in that specific way, a namespace `Goodcity::Operations` would also work nicely. I'm open to any change.

The important thing is that we group them and maintain them as a whole 💪